tools/deploy_config.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
It would be nice to keep everything related to deploying in `/deploy` dir. Can we move this to `deploy/deploy_config`?

Reply:
sure